common/web/keyboard-processor/src/text/keyboardProcessor.ts

```diff
@@ -235,6 +235,7 @@ export default class KeyboardProcessor extends EventEmitter<EventMap> {
       matchBehavior = this.defaultRuleBehavior(keyEvent, outputTarget, false);
       matchBehavior.triggerKeyDefault = true;
       // Force a single `deleteLeft`.
+      // @ts-ignore // In order to force a value override.
       matchBehavior.transcription.transform.deleteLeft = 1;
```

Comment:
`deleteLeft` is marked readonly.

This is the final post-processing step for keyboard rules, and it's far less tedious to override the value than it is to reconstruct the whole object hierarchy from scratch.

Reply:
```suggestion
      // @ts-ignore // force value override, because deleteLeft is marked readonly.
      matchBehavior.transcription.transform.deleteLeft = 1;
```

OK I guess ... feels a little dirty but we can live with it!